base/sort.jl

```diff
@@ -735,7 +735,7 @@ end
 
 # For AbstractVector{Bool}, counting sort is always best.
 # This is an implementation of counting sort specialized for Bools.
-# Accepts unused buffer to avoid method ambiguity.
+# Accepts unused scratch space to avoid method ambiguity.
```

Comment:
Why does this say "unused"? It seems to be used?

Reply:
It is used in the `sort!(::AbstractVector, ...)` case, but this comment refers to the `sort!(::AbstractVector{Bool}, ...)` method which does not use the scratch space/buffer